### ISSUE TITLE
Name field does not render in preview email notification, nor does it get send in the email notification

### ISSUE THREAD

**Issue by DavidLaridon** [verbb/formie#1944]
### Describe the bug

I made a simple contact form, with a name, email and multiline field. When i try to add the name field to an email notification it doesn't give an example in the preview. It also doesn't get send in the email notification. Its the same if i set the field to use multiple fields (firstname, lastname,...). The issue also remains when making new forms or new email notifications.

All other fields seem to be fine though, not sure what could cause this.

I do use a template for my notification, but it's nothing special:
```
<!doctype html>
<html>
  <head>
    <style>
      @import url("https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap");
    </style>
  </head>

  <body style="font-family: 'Inter'; background-color: '#fff'; padding: 10px 10px; color: '#303241">
    <div>
      {{ contentHtml }}
    </div>
  </body>
</html>
```
### Steps to reproduce

1. Make a form
2. Add a name field
3. Make a notification
4. Add the name field to the notification

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

5.2.1

### Plugin version

3.0.0-beta.15

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.15"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.16](https://github.com/verbb/formie/releases/tag/3.0.0-beta.16)
